src/Raven.Studio/typescript/commands/maintenance/getAdminLogsMicrosoftStateCommand.ts

```diff
@@ -0,0 +1,20 @@
+import commandBase = require("commands/commandBase");
+import endpoints = require("endpoints");
+import Dictionary = System.Collections.Generic.Dictionary;
+
+interface GetAdminLogsMicrosoftStateCommandResult {
```

Comment:
if this type is available on server side, let's use typing generator.

Reply:
on server side it is DynamicJsonValue

Reply:
so let's leave it as it is.